.gitignore

```diff
@@ -4,6 +4,7 @@ coverage/
 dist
 node_modules/
 *.log
+.env
```

Comment:
AFAIR create react app supports `.env.dev` and `.env.prod` files, we might want to rename `.env-example` to `.env.dev`. Then production settings can be overwritten by creating `.env` or `.env.prod` file.

Reply:
Here is the doc for env variables with parcel https://parceljs.org/env.html

Reply:
I have no strong opinion. Never really played with anything else other than the plain .env. Is the docker-compose setting `NODE_ENV` ?

Reply:
Oh, cool, I didn't know we are using parcel. Yeah, I guess we should keep the `.env` file then (instead of `.env-example`) and let people create `.env.local` to override it.

Reply:
`docker-compose` is just collecting images (or builds them based on `Dockerfile`). The `NODE_ENV` should be set automatically to `production` when you simply `npm build`. `.env` seems like it's the most low-level, so any other file (`.local`, .development` and `.development.local`) overrides any settings, hence I suggested we keep `.env` in the repo and let people override the vars in other files.

Reply:
Agreed. that's a reflex I have to add the .env to gitignore to prevent leaking secrets.. but it obviously doesn't make sense for front-end.